substrate/frame/support/src/traits/tokens/fungible/mod.rs

```diff
@@ -58,13 +58,17 @@
 //!   3 holds for 100 units, the account can spend its funds for any reason down to 300 units, at
 //!   which point the holds will start to come into play.
 //!
-//! - **Frozen Balance**: A freeze on a specified amount of an account's free balance until a
-//!   specified block number.
+//! - **Frozen Balance**: A freeze on a specified amount of an account's balance. This amount is 
+//!   set aside and cannot be spent by the account holder.  
 //!
 //!   Multiple freezes always operate over the same funds, so they "overlay" rather than
 //!   "stack". This means that if an account has 3 freezes for 100 units, the account can spend its
 //!   funds for any reason down to 100 units, at which point the freezes will start to come into
 //!   play.
+//!   
+//!   It's important to note that the frozen balance can exceed the total balance of the account.
+//!   In such cases, the account cannot spend any fund and the excess frozen amount doesn't 
+//!   actually belong to the account, only the lock does.
```

Comment:
```suggestion
//!   It's important to note that the frozen balance can exceed the total balance of the account.
//!   This can be used in scenarios where you would want to keep an account's balance frozen, independent of the specific amount of balance they have. For example, by setting the users frozen balance to `Balance::MAX`, the user will not be able to transfer any funds in the account. Developers should be conscious to use the `balance_freezable` function to determine the amount of available balance of the user which can become frozen.
```